apps/shipments/geojson.py

```diff
@@ -84,18 +94,33 @@ def __init__(self, point):
             self.source = point['source'] if 'source' in point else None
 
         except IndexError:
+            LOG.error(f'Device tracking index Error {IndexError}.')
+            log_metric('transmission.error', tags={'method': 'device_tracking_index_error'})
+
             raise InvalidTrackingPointError(f"Invalid Coordinates format from device")
 
         except KeyError as key_error:
+            LOG.error(f'Device tracking key Error {key_error}.')
+            log_metric('transmission.error', tags={'method': 'device_tracking_key_error'})
+
             raise InvalidTrackingPointError(f"Missing field {key_error} in tracking data from device")
 
     def as_point(self):
+        LOG.debug(f'Device tracking as_point.')
+        log_metric('transmission.info', tags={'method': 'as_point'})
+
         try:
             return Point((self.lon, self.lat))
         except Exception:
+            LOG.error(f'Device tracking as_point exception {Exception}.')
```

Comment:
This isn't going to print out anything but the Exception class. If you want to capture the details of the exception, you have to catch it like this:
`except Exception as ex:`
and then print out `ex`.